contributors/guide/first-contribution.md

```diff
@@ -57,7 +57,7 @@ When you've found an issue to work on, you can assign it to yourself.
 ## Learn about SIGs
 
 Some repositories in the Kubernetes Organization are owned by 
-[Special Interest Groups], or SIGs.
+[Special Interest Groups](./sig-list.md), or SIGs.
```

Comment:
```suggestion
[Special Interest Groups][sl], or SIGs.
```
`./sig-list.md` would map to a file in the same directory and still break.

The link is there, but at the bottom of the doc as a reference link (`[sl]`)

```
[sl]: /sig-list.md
```